modules/lang-painless/src/main/java/org/elasticsearch/painless/GenericElasticsearchScript.java

```diff
@@ -16,10 +16,20 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+
 package org.elasticsearch.painless;
 
+import org.elasticsearch.index.fielddata.ScriptDocValues;
+
+import java.util.Map;
+
 /**
- * Marker interface that a generated {@link Executable} uses the {@code _score} value
+ * Generic script interface that all scripts that elasticsearch uses implement.
```

Comment:
I think the comment is accurate until we get contexts. Maybe it should read "Generic script interface that Painless implements for all Elasticsearch scripts." or something like that. One day we'll be able to remove this entirely. I think.